satnav/utils/parse-sqs-event.js

```diff
@@ -18,17 +18,15 @@
     )
   }
 
-  const carId = sqsEvent.Records[0].body
-  const info = carId.match(/([^/]+)\/([^/]+)\/(.+)/)
-  if (!info) {
+  const body = sqsEvent.Records[0].body
```

Comment:
do we end up with a lib module at some point? I wish we didn't have to do this boilerplate at all!

Reply:
Yes, I think we should. I will create issue